CHANGES.md

```diff
@@ -15,6 +15,11 @@ development source code and as such may not be routinely kept up to date.
 
 ## Improvements
 
+* `nextstrain setup <pathogen>` and `nextstrain version --pathogens` now list
+  the available workflows for a pathogen if the pathogen lists the workflows
+  in the it's top-level `nextstrain-pathogen.yaml` file.
```

Comment:
Small typo:

```suggestion
  in the top-level `nextstrain-pathogen.yaml` file.
```

Reply:
Thanks! Fixed in https://github.com/nextstrain/cli/commit/bb36adf07a4a65dda4c2b89a154ee350a1c4253a